ChangeLog.md

```diff
@@ -1,6 +1,22 @@
 
 # Change Log
 
+## Version 10.26.0
```

Comment:
Let's add a new features section, with a point about adding support for the package managers, and specify which ones are now supported.

Reply:
sure will add it